### PR TITLE
FEAT: Added Image component

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-14 15-34-30

### CODE REVIEW

File: Components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+:root {
+    --black: #000000;
+    --medium-font-size: medium;
+    --large-font-size: large;
+    --larger-font-size: larger;
+    --bolder-font-weight: bolder;
```
Comment (viru2001) [inline Components/Image/image.css]: you can give font size in relative units

File: Components/Image/image.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+* {
+    box-sizing: border-box;
+}
+
+span {
+    font-weight: var(--bolder-font-weight);
+}
```
Comment (viru2001) [inline Components/Image/image.css]: you should give class name/ id while using span. don't style span directly

File: Components/Image/image.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                    They expand upto the width of the container they are in. To use this image component, name the class
+                    as <span>responsive-image</span> for your image element.
+                </div>
+                <div class="responsive-image-container">
+                    <img class="responsive-image" src="/Assets/Images/Forest-landscape.jpg" />
+                </div>
+                <div class="code-frame">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28184%2C176%2C141%2C0%29&t=3024-night&wt=none&l=auto&width=680&ds=true&dsyoff=0px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=4x&wm=false&code=%2520%253Cimg%2520class%253D%2522responsive-image%2522%2520src%253D%2522image-path-link%2522%252F%253E"
+                        style="width: 100%; height: 190px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (viru2001) [inline Components/Image/image.html]: you should avoid inline styles

**Review by viru2001 — CHANGES_REQUESTED** [review]
I suggested some changes , Everything else looks good

File: Components/Image/image.css (hunk 1 above)
Comment (yashpatel1797) [inline Components/Image/image.css]: i suggest instead of using medium, larger you can use value.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to me.

File: Components/Image/image.css (hunk 1 above)
Comment (kotesh-arya) [reply Components/Image/image.css]: Done

File: Components/Image/image.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/Image/image.css]: Done 👍🏻

File: Components/Image/image.css (hunk 1 above)
Comment (kotesh-arya) [reply Components/Image/image.css]: Yeah, I have updated them 👍🏻

File: Components/Image/image.html (hunk 3 above)
Comment (kotesh-arya) [reply Components/Image/image.html]: Yeah, I've avoided them. Thanks for the suggestion 👍🏻

**Pull request merged**